inspect_byproducts.py

```diff
@@ -110,24 +112,24 @@ def parse_args():
 
     parser.usage = ("\n"
                     "%(prog)s --link <path to link metadata>\n{0}"
-                    "[--type of field to be inspected (--stdout | --stderr)]\n{0}"
-                    "[--operator (--is | --is not | --contains | --"
-                    "contains  not)]\n{0}"
+                    "[--type (--stdout | --stderr)]\n{0}"
+                    "[--operator (--is | --is-not | --contains | --"
+                    "contains-not)]\n{0}"
                     "string\n\n"
                     .format(lpad))
 
     in_toto_args = parser.add_argument_group("in-toto-inspection options")
 
     in_toto_args.add_argument("-l", "--link", type=str, required=True,
-                              help="Link metadata file to use for inspection "
-                              "of the step")
+                              help="Link metadata file to use for inspection"
+                              " of the step")
 
     in_toto_args.add_argument("-t", "--type", choices=['stdout', 'stderr'],
-                              type=str, required=True, help="Whether stdout or "
-                              "stderr is a byproduct")
+                              type=str, required=True, help="Whether stdout or"
+                              " stderr is a byproduct")
 
-    in_toto_args.add_argument("-o", "--operator", choices=['is', 'is not',
-                              'contains', 'contains not'], type=str,
+    in_toto_args.add_argument("-o", "--operator", choices=['is', 'is-not',
+                              'contains', 'contains-not'], type=str,
```

Comment:
wouldn't this be `%(prog)s --link whatever.link --operator ( is | is-not | contains | contains-not)` or so?